src/dgnss_management.c

```diff
@@ -64,10 +36,10 @@ void make_measurements(u8 num_double_diffs, const sdiff_t *sdiffs, double *raw_m
   DEBUG_EXIT();
 }
 
-static bool prns_match(const u8 *old_non_ref_prns, u8 num_non_ref_sdiffs,
+static bool prns_match(const dgnss_state_t *dgs, const u8 *old_non_ref_prns, u8 num_non_ref_sdiffs,
```

Comment:
this doesn't need the whole state, just the sats management. Why not pass that in instead?